src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -48,11 +48,11 @@ class ItemRecognition extends Component {
     this.model.predict(img).then(async item => {
       if (
         item.value > ML_THRESHOLD &&
-        item.id !== '' &&
+        item.id !== 'unknown' &&
         !this.props.prediction
       ) {
         await this.addTrainingImage(img.src, item.id);
```

Comment:
Is it necessary to wait for the training image to be added?

Reply:
no